Userland/Libraries/LibSoftGPU/ImageFormat.h

```diff
@@ -115,6 +140,13 @@ inline static void pack_color(FloatVector4 const& color, void* ptr, ImageFormat
     case ImageFormat::RGB565:
         *reinterpret_cast<u16*>(ptr) = (r & 0x1f) | ((g & 0x3f) << 5) | ((b & 0x1f) << 11);
         return;
+    case ImageFormat::L8:
+        *reinterpret_cast<u8*>(ptr) = r;
```

Comment:
I'm not sure if it's okay to just take the red value here. For luminance (+ alpha), `r`, `g` and `b` should be the same though.

Reply:
Looks fine to me, and this doc seems to imply that this is the case: https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture.txt

The image also looks correct; it'd be jarring if this was messed up.